### PR TITLE
Make admin_settings_aws_region Moodle 2.6-3.1 compatible

### PR DESCRIPTION
**Pull request by golenkovm**
This is a fix for #19

### CODE REVIEW

File: templates/setting_list.mustache [hunk 1]
```diff
@@ -41,7 +41,7 @@
     Setting configtext.
 }}
 <div class="form-text defaultsnext">
-    <input list="{{name}}" name="{{name}}" value="{{value}}" size="{{size}}" id="{{id}}" class="form-control {{#forceltr}}text-ltr{{/forceltr}}">
+    <input type="text" list="{{name}}" name="{{name}}" value="{{value}}" size="{{size}}" id="{{id}}" class="form-control {{#forceltr}}text-ltr{{/forceltr}}"/>
```
Comment (golenkovm) [inline templates/setting_list.mustache:44]: This change is needed to fix input appearance:

image

File: classes/admin_settings_aws_region.php [hunk 2]
```diff
@@ -61,16 +61,35 @@ public function output_html($data, $query='') {
...
+        if (!empty($CFG->branch) && $CFG->branch > 31) {
+            $context = (object) [
+                'size' => $this->size,
+                'id' => $this->get_id(),
+                'name' => $this->get_full_name(),
+                'value' => $data,
+                'forceltr' => $this->get_force_ltr(),
+                'options' => $options,
+            ];
+            $element = $OUTPUT->render_from_template('local_aws/setting_list', $context);
```
Comment (brendanheywood) [inline classes/admin_settings_aws_region.php:73]: as discussed, not much point in supporting two code paths that do the same thing, lets just ditch the mustache template and do it the html_writer way for everyone

Comment (golenkovm) [reply classes/admin_settings_aws_region.php]: Done

**Comment by golenkovm** [comment]
Moodle 2.6:

image

**Pull request merged**